pages/api/submissions/index.ts

```diff
@@ -0,0 +1,25 @@
+import forms from 'data/flexibleForms/forms';
+import { NextApiRequest, NextApiResponse } from 'next';
+import { addCase } from 'lib/cases';
+
+const handler = async (
+  req: NextApiRequest,
+  res: NextApiResponse
+): Promise<void> => {
+  if (req.method === 'POST') {
+    const { socialCareId, formId } = req.body;
+
+    // TODO: start a new submission here
+
+    // TODO: send back new case object
+    res.json({
+      id: 1,
+    });
+  } else {
+    res.json({
```

Comment:
This would return this JSON if I sent a `DELETE` to this endpoint? We should probably be more explicit to only handle the HTTP methods we want, and 404 the rest.

Reply:
this is actually closer to next.js's suggestions in their docs - the switch statements we have in other places i find pretty verbose and harder to read, but eh 🤷‍♀️

Reply:
done